src/test/admin-server/index.ts

```diff
@@ -1,25 +1,51 @@
 import { ApolloServer } from 'apollo-server-express';
-import { buildFederatedSchema } from '@apollo/federation';
-import { ApolloServerPluginLandingPageDisabled } from 'apollo-server-core';
+import { buildSubgraphSchema } from '@apollo/federation';
+import {
+  ApolloServerPluginInlineTraceDisabled,
+  ApolloServerPluginLandingPageDisabled,
+  ApolloServerPluginUsageReportingDisabled,
+} from 'apollo-server-core';
 import { typeDefsAdmin } from '../../typeDefs';
 import { resolvers as adminResolvers } from '../../admin/resolvers';
 import { client } from '../../database/client';
+import { ContextManager } from '../../admin/context';
+import s3 from '../../aws/s3';
 
 // Export this separately so that it can be used in Apollo integration tests
 export const db = client();
 
+// Only using one value from MozillaAccessGroup enum in Pocket Shared Data
+export const COLLECTION_CURATOR_FULL =
+  'mozilliansorg_pocket_collection_curator_full';
```

Comment:
i don't know if this is a nit-pick or not - i feel like it's a little messy to keep this enum definition here. we're going to need the read only value as well for queries, and while i guess we could keep them both in here, it just feels...messy. would like to know how others feel.

cc @Herraj

Reply:
I agree, it was not a good spot for it and it was defined once more in the context. We don't really have any shared data here and the only `types.ts` file is for database resolvers, so I created a `src/shared/constants.ts` file and placed the constant there.